objcgen/objcgenerator-postprocessor.cs

```diff
@@ -24,6 +25,11 @@ protected IEnumerable<ProcessedMethod> PostProcessMethods (IEnumerable<MethodInf
 
 				if (method.IsSpecialName && method.IsStatic && method.Name.StartsWith ("op_", StringComparison.Ordinal))
 					processedMethod.IsOperator = true;
+				
+				if (operatorToIgnore.Contains (method)) {
+					processedMethod.Ignore = true;
```

Comment:
We could do that as well. I was unsure if there was a case where we'd want the information around but not during generation. (A depends on some info on B or the like).

Since neither of us can think of a case, let's go with the simple version.